input/images-source/overview-app-launch.plantuml

```diff
@@ -3,27 +3,27 @@ participant App
 participant "EHR with Authorization Server" as EHR
 participant "FHIR Server" as FHIR
 
-opt  Precondition: Client Registration
+opt  1. Precondition: Client Registration
```

Comment:
These numbers don't line up vertically and don't make the diagram clearer for me. I'm hesitant to incorporate them here, especially if we're not going to incorporate them into the section titles below

Reply:
There is a numbered list next to a sequence diagram - the obvious question the reader has is how are they related.   I know its not pretty,. the alternative is to renumber the list to correspond to the sequentially number sequence diagram.

Reply:
An alternative might be to group request/response arrows more directly so that numbers aren't needed, or aren't needed on both the request and the response. Or just change the numbers to bullets in the list below.

Reply:
concede the diagram is worse with numbering